test

```diff
@@ -0,0 +1,6 @@
+You are now connected to database "nc_games_test" as user "srvadmin".
+ review_id |       title       |     category     |   designer   |   owner    |          review_body           |                              review_img_url                               |       created_at        | votes | comment_count 
```

Comment:
happy for this to be used but please do add this to your .gitignore as this does not need to be on github.